### ISSUE TITLE
Add some moving elements to improve the feeling of movement

### ISSUE THREAD

**Issue by Sogolumbo** [bydariogamer/beatgame#5]
Add a background which moves at a lower speed than the foregroud.
Add texture to the bottom bar and make the texture move with the obstacles.

**Comment by bydariogamer** [comment]
I PROMISE I WAS GOING TO. It was in the first draft of the game... The problem is I made it in just some days for a game jam and I had to decide: the songs working or the fancy background. I will add it :)

**Comment by bydariogamer** [comment]
I will do it right now. What about adding some kind of stars 🤔 ?

**Comment by Sogolumbo** [comment]
Stars sound like a good idea. It already feels spacy anyways :)

**Comment by bydariogamer** [comment]
I have no idea for the ground however... The game jam was about high tech so I had planned to add that kind of elements, but now it seems definitely "spacy" yes 😂 
I am not sure about what to put in the ground

**Comment by Sogolumbo** [comment]
Maybe mimic the moon surface by putting grey circles/ellipses in the ground.

**Comment by bydariogamer** [comment]
Good idea... I may do multiple planets backgrounds. I commit an hour ago a high-score record system (the save function in player.py) that could unlock stuff like changing background or character (current one is not bad, but some new ones won't to any damage).

**Comment by Sogolumbo** [comment]
I have one more idea:
We could make the obstacles drop a shadow onto the floor (slightly diagonally).

**Comment by Sogolumbo** [comment]
I like the stars!
Some of the stars seem to wander a bit, are they supposed to do that? If not I guess you introduced too much rounding errors, maybe switch to floats if needed.

**Comment by bydariogamer** [comment]
I tried to make them a bit 'not that coordinated' so the sky is a bit more realistic... However, I found that the randomness of sky creates an 'optical illusion' then the player stops moving... I felt it was still moving a bit but it is not... some kind of human eye or brain problem 😋

**Comment by bydariogamer** [comment]
Should the element in the ground move at a difference speed from the rest? (I mean, less than obstacles, but more than the stars)

**Comment by Sogolumbo** [comment]
I don't know. I'd say just try 100% and maybe 80% of the obstacle speed and see what's looking best.
